### PR TITLE
explain usage of custom values for services in more detail

### PR DESCRIPTION
**Pull request by andreas-kupries**
ref https://github.com/epinio/epinio/pull/2485

Documentation for the more complex forms of custom values for services, now that thy are supported

### CODE REVIEW

**Comment by andreas-kupries** [comment]
Honestly, I would like to have a table, with the three things side by side.
That requires HTML however IIRC, as MD tables do not support complex cells.

**Comment by andreas-kupries** [comment]
And we have tables now, after running into the wall of docasaurus MDX/JSX which is not plain MD.
Thanks to @nunix for the help with that.

File: docs/howtos/using_custom_service_values.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+the operators gave them.
+
+Operators are taught how to specify user settings based on the form of the field in the
+`values.yaml` file of the underlying service helm chart.
+
+Users are taught how to set values for the user settings of a service.
+
+# General information
+
+For operators, see [Creating A Custom Service](using_custom_appcharts.md) for the general syntax of
```
[the remark this answers is not in the document]
Comment (andreas-kupries) [reply docs/howtos/using_custom_service_values.md:20]: Actually the link should to be Service, which in turn points to the Application location.

[the remark this answers is not in the document]
Comment (andreas-kupries) [reply docs/howtos/using_custom_service_values.md:20]: I will fix that, and then trial a different way of writing the table Nuno wrote to me in slack.

[the remark this answers is not in the document]
Comment (andreas-kupries) [reply docs/howtos/using_custom_service_values.md]: Commit e98a89e5d6530a0cc21305ddd6c3835e10d20ec8

**Review by thehejik — APPROVED** [review]
LGTM, I believe it covers most of the custom value data structures now.

**Pull request merged**